cmd/kube-version-change/version.go

```diff
@@ -28,6 +28,8 @@ import (
 	"runtime"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1"
```

Comment:
Thanks. Yes, now that we add api/latest in the imports, we can remove imports of v1/v1beta3 now.